Core/Steps/PipelineSteps/BranchFromHotfixStep.cs

```diff
@@ -92,6 +92,6 @@ private SemanticVersion GetCurrentHotfixVersion (bool startReleasePhase = false)
     var mostRecent = _semanticVersionedGitRepository.GetMostRecentHotfixVersion();
 
     var possibleVersions = mostRecent.GetCurrentPossibleVersionsHotfix();
-    return _inputReader.ReadVersionChoice("Please choose next Version:", possibleVersions);
+    return _inputReader.ReadVersionChoice("Please choose version to be released:", possibleVersions);
```

Comment:
The other steps also have prompts with the text "Choose next version (open JIRA issues get moved there):". Those are not relevant to this issue? Would it make sense to clean up the wording here as well or is this out of scope?

Reply:
That is not relevant in this issue, those already portrait the intention correctly.